pandas/tests/groupby/test_value_counts.py

```diff
@@ -79,3 +79,28 @@ def rebuild_index(df):
     # have to sort on index because of unstable sort on values
     left, right = map(rebuild_index, (left, right))  # xref GH9212
     tm.assert_series_equal(left.sort_index(), right.sort_index())
+
+
+@pytest.mark.parametrize("freq", ["1D", "2D", "1W", "1Y"])
+@pytest.mark.parametrize("size", [100, 1000])
+@pytest.mark.parametrize("frac", [0.1, 0.5, 1])
+def test_series_groupby_value_counts_with_grouper(freq, size, frac):
+    np.random.seed(42)
```

Comment:
Refer to the GH issue number in a comment.

Reply:
@TomAugspurger  OK, I added it. After you give me replies about other conversations, I'll push it. However, currently, Travis CI repeatedly gives me fails as follow:
```
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received
The build has been terminated
```
do you know what's this?

Reply:
Should be fixed if you merge master before pushing.

Reply:
ok, thank you.